projects/microphysics/scripts/prognostic_run.py

```diff
@@ -83,15 +67,16 @@ def get_env(args):
 
 config = dacite.from_dict(HighLevelConfig, config)
 config.namelist["gfs_physics_nml"]["emulate_zc_microphysics"] = args.online
-config = config.to_fv3config()
+if args.model:
```

Comment:
Should this be allowed to overwrite an existing model if it exists in the config? If yes, can you at least warn? If no can you raise an error?

Reply:
This is intended, I had debated removing the flag altogether earlier. It would be useful to pipeline training/evaluation, but I'll just remove it to keep it simple.